qemu/tests/cpu_device_hotpluggable.py

```diff
@@ -132,10 +133,36 @@ def sub_pause_resume():
         pluggable_vcpu = -(vcpus_count * len(pluggable_vcpu_dev))
     expected_vcpus = vm.get_cpu_count() + pluggable_vcpu
 
+    # Add a workaround for win2016 guest to solve the issue that
+    # https://support.huawei.com/enterprise/zh/doc/EDOC1100034211/5ba99a60.
+    # Add steps for install driver of "HID Button over Interrupt Driver".
```

Comment:
There are other cpu hotplug test cases need this workaround. Suggest to wrap it in a single function and invoke it in related test cases.

Reply:
Hi yuhuang, I checked the privider directory, only cpu_utils.py related this case, but it doesn't suitable setting this function in. It's also as a case level setting, seems not suitable set in avocado vt as well, How do you think create a new file for this workaround under provider directory? thus convenient other cpu hotplug test cases to call it and also can avoid case call case. cc @PaulYuuu

Reply:
Sounds good to me.

Reply:
Or do you have any better advice, thanks  a lot~@huangyum @PaulYuuu

Reply:
Updated, thanks~